### ISSUE TITLE
Syntax Error(ruby2.3.0)

### ISSUE THREAD

**Issue by ore-public** [khotta/color_echo#1]
``` ruby
require 'color_echo'

CE.ch :green
puts "Inported: a.json, count: 1111"
CE.off
```

```
/Users/hoge/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/color_echo-2.0.1/lib/color_echo/variables.rb:33:in `eval': (eval):1: `@@<' is not allowed as a class variable name (SyntaxError)
    from /Users/hoge/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/color_echo-2.0.1/lib/color_echo/variables.rb:33:in `block in <module:CE>'
    from /Users/hoge/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/color_echo-2.0.1/lib/color_echo/override.rb:15:in `puts'
    from main.rb:4:in `<main>'
```

**Comment by khotta** [comment]
Hi, Thanks for your inquiry.
The problem be caused by the difference of return value about Kernel.#caller between '>= 2.3.0' and under it.
I've fixed this issue on color_echo-2.0.3.
But I think this way isn't rather smart. I'll try to find an other way of this problem.
I hope you to continue to use color_echo :)

**Comment by ore-public** [comment]
thanks

It was successfully After you verify with the new version.
